packages/infra/lib/hl7-notification-routing-stack/mllp.ts

```diff
@@ -54,21 +58,22 @@ export class MllpStack extends cdk.NestedStack {
       memoryLimitMiB: fargateMemoryLimitMiB,
       desiredCount: fargateTaskCountMin,
       taskImageOptions: {
-        image: ecs.ContainerImage.fromDockerImageAsset(dockerImage),
+        image: ecs.ContainerImage.fromEcrRepository(ecrRepo, "latest"),
```

Comment:
Related: https://github.com/metriport/metriport-internal/issues/1288

I'll update the ticket to also consider this service.

Reply:
done

Reply:
Yeah I remember us discussing the latest task being a pain in the ass. I figured this should just be a decoupled change from this PR that gets made to both API and MLLP server.